packages/devtools_app/lib/src/shared/screen.dart

```diff
@@ -55,6 +56,8 @@ enum ScreenMetaData {
     icon: Octicons.package,
     requiresDartVm: true,
     tutorialVideoTimestamp: '?t=420',
+    // ignore: avoid_redundant_argument_values, false positive
+    worksWithOfflineData: kMemoryOfflineExperiment,
```

Comment:
use `FeatureFlags.memoryOffline` instead of the "k" constant

Reply:
Enums require constants. FeatureFlags.memoryOffline is not constant.

Reply:
make memoryOffline constant and then you should be able to use it.